### PR TITLE
added client edit option

### PR DESCRIPTION
**Pull request by mukulsaini**
@championswimmer  Issue #24 resolved. Please check

### CODE REVIEW

File: routers/api/clients.js [hunk 1]
```diff
@@ -40,5 +40,35 @@ router.post('/add', function (req, res) {
     })
 });
 
+router.put('/edit/:id', function(req, res){
```
Comment (championswimmer) [inline routers/api/clients.js]: would prefer using only GET and POST requests throughout

File: routers/api/clients.js [hunk 2]
```diff
@@ -40,5 +40,35 @@ router.post('/add', function (req, res) {
...
+    if(!req.user){
+        return res.send("Only logged in users can edit clients")
```
Comment (championswimmer) [inline routers/api/clients.js]: Use connect-ensure-loggedin (like other places) to simple redirect user to login page

File: routers/api/clients.js (hunk 1 above)
Comment (mukulsaini) [inline routers/api/clients.js]: @championswimmer  ok. I have done the required changes in the latest commit.

File: routers/api/clients.js (hunk 2 above)
Comment (mukulsaini) [inline routers/api/clients.js]: done :+1:

**Comment by championswimmer** [comment]
Need to also add the frontend part for it. A hbs page for editing client. And on that page, clicking the 'submit' button should hit this endpoint.

**Comment by mukulsaini** [comment]
@championswimmer  I have added frontend part for edit option, you can review it.

**Comment by championswimmer** [comment]
@mukulsaini 
This had a lot of errors https://github.com/coding-blocks/oneauth/commit/c7e7fc690aef4ef5c1cbb9de683ae2b8896db338 

In future I will not tolerate PRs that you have **NOT tested at all on your end** 

You can go ahead and claim your bounty on boss.codingblocks.com

**Comment by mukulsaini** [comment]
sorry, I will not repeat such mistake again.

**Pull request merged**